dash/dash-eme.js

```diff
@@ -1,213 +0,0 @@
-function prefixedAttributeName(obj, suffix, opt_preprefix) {
```

Comment:
I removed the experimental DASH player we used in this project long ago to test for native Media Source compatibility. We'll be converting contrib-hls to use real media sources soon which will be a much more robust compatibility check.